definitions/ext-rservice/definition.yml

```diff
@@ -0,0 +1,78 @@
+domain: EXT
+type: RSERVICE
+synthesis:
+  rules:
+  - compositeIdentifier: 
+      separator: ":"
+      attributes:
+      - rfc190_environment
+      - application
+    name: application
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: metricName
+      present: true
+    - attribute: application
+      present: true
```

Comment:
This one and `rfc190_environment` you can remove.

We already implicitly require them to be present, given they are required to build the GUID and the name of the entity.

That should put you already below the 10 limit.